fields/types/boolean/test/explorer.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
 	Field: require('../BooleanField'),
 	Filter: require('../BooleanFilter'),
+	readme: require('fs').readFileSync('./fields/types/boolean/Readme.md', 'utf8'),
```

Comment:
Well, for webpack you define the loader for `.md` files to be `raw-loader`, and then `import readme from '../Readme.md'` will do what you want.

For browserify, looks like you need to add https://github.com/JohnPostlethwait/stringify